notice/1026/2016-28718.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<notice xmlns="eregs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="eregs http://cfpb.github.io/regulations-schema/src/eregs.xsd">
+  <fdsys>
+    <cfrTitleNum>12</cfrTitleNum>
+    <cfrTitleText>Banks and Banking</cfrTitleText>
+    <volume>8</volume>
+    <date>2016-11-30</date>
+    <originalDate>2012-01-01</originalDate>
+    <title>CONSUMER LEASING</title>
+  </fdsys>
+  <preamble>
+    <agency>Bureau of Consumer Financial Protection</agency>
+    <regLetter>Z</regLetter>
+    <cfr>
+      <title>12</title>
+      <section>1026</section>
+    </cfr>
+    <documentNumber>2016-28718</documentNumber>
+    <effectiveDate>2017-01-01</effectiveDate>
+    <federalRegisterURL>https://www.federalregister.gov/documents/2016/11/30/2016-28718/truth-in-lending-regulation-z</federalRegisterURL>
+  </preamble>
+  <changeset leftDocumentNumber="2016-30730" leftEffectiveDate="2017-01-01" rightDocumentNumber="2016-28718">
+    <change label="1026-3-b-Interp" operation="modified">
+      <interpParagraph label="1026-3-b-Interp" marker="" target="1026-3-b">
+        <content></content>
+        <interpParagraph label="1026-3-b-Interp-1" marker="1.">
+          <content>For purposes of &#167; 1026.3(b), the threshold amount in effect during a particular period is the amount stated in comment 3(b)-3 below for that period. The threshold amount is adjusted effective January 1 of each year by any annual percentage increase in the Consumer Price Index for Urban Wage Earners and Clerical Workers (CPI-W) that was in effect on the preceding June 1. Comment 3(b)-3 will be amended to provide the threshold amount for the upcoming year after the annual percentage change in the CPI-W that was in effect on June 1 becomes available. Any increase in the threshold amount will be rounded to the nearest $100 increment. For example, if the annual percentage increase in the CPI-W would result in a $950 increase in the threshold amount, the threshold amount will be increased by $1,000. However, if the annual percentage increase in the CPI-W would result in a $949 increase in the threshold amount, the threshold amount will be increased by $900.</content>
```

Comment:
Just curious, why are there no title elements here, e.g. from here:

![image](https://user-images.githubusercontent.com/654645/29732286-f3c0ee70-89b5-11e7-8c21-c0394be098b9.png)

Reply:
I must have just forgot them. Thanks, they're in now.